backend/app/models/enrollment.rb

```diff
@@ -538,4 +538,22 @@ def submit_validation
         message: "Vous devez confirmer avoir informé le DPD de votre organisation avant de continuer")
     end
   end
+
+  def hubee_validation
```

Comment:
Perhaps to be more precise, this variable should be called "no_hubee_certdc_validation".